pyflakes/checker.py

```diff
@@ -1699,7 +1699,34 @@ def CALL(self, node):
                     with self._enter_annotation():
                         self.handleNode(keyword.value, node)
 
-        self.handleChildren(node)
+        if any(_is_typing(node.func, nt, self.scopeStack)
+                for nt in ("TypedDict", "NamedTuple")):
```

Comment:
this `any` no longer buys you anything because you have separate cases below (it already was a little bit of an over-DRYing before -- two things isn't repeating yourself)

Reply:
I would leave it in case of future new named type and because it would repeat 
```python
with self._enter_annotation(AnnotationState.NONE):
    self.handleChildren(node)
```
 and
```python
for a in attributes:
      if isinstance(a, ast.Str):
          with self._enter_annotation():
              self.handleNode(a, node)
```

Reply:
I would prefer the reduced complexity, thanks -- over-DRYing and pre-DRYing make code harder to maintain